packages/review/src/components/review-placeholder-with-rating.tsx

```diff
@@ -68,6 +70,7 @@ export function ReviewsPlaceholder({
         navigateReviewList({
           resourceId,
           resourceType,
+          hasMedia: false,
```

Comment:
```suggestion
          hasMedia,
```

hasMedia prop 전달하는 거 아니라 false로 고정하는 거 맞나요?

Reply:
네 여긴 고정이 맞아요 !